removed none debian files from the install target
Index: pcapsipdump-0.2/Makefile
===================================================================
--- pcapsipdump-0.2.orig/Makefile
+++ pcapsipdump-0.2/Makefile
@@ -22,7 +22,5 @@ clean:
 
 install:
 	install pcapsipdump ${DESTDIR}/usr/sbin/pcapsipdump
-	install redhat/pcapsipdump.init ${DESTDIR}/etc/rc.d/init.d/pcapsipdump
-	install redhat/pcapsipdump.sysconfig ${DESTDIR}/etc/sysconfig/pcapsipdump
-	mkdir -p ${DESTDIR}/var/spool/pcapsipdump
-	chmod 0700 ${DESTDIR}/var/spool/pcapsipdump
+	install -m0700 -d ${DESTDIR}/var/spool/pcapsipdump
+#	chmod 0700 ${DESTDIR}/var/spool/pcapsipdump
